lib/generators/gamora/templates/gamora.rb

```diff
@@ -22,4 +22,5 @@
   # config.allow_create = true
   # config.userinfo_cache_expires_in = 1.minute
   # config.introspect_cache_expires_in = 0.seconds
+  # config.authorization_method = -> (user) { user.present? }
```

Comment:
IMO to better show the intention of the feature, I will sample here with something like:

```
# config.authorization_method = -> (user) { user.teacher? }
```

or something that indicates better that we are looking for authorization

Reply:
Done, please take another look.